lib/relation-definition.js

```diff
@@ -600,7 +649,7 @@ RelationDefinition.hasMany = function hasMany(modelFrom, modelTo, params) {
   var discriminator, polymorphic;
 
   if (params.polymorphic) {
-    polymorphic = polymorphicParams(params.polymorphic);
+    polymorphic = polymorphicParams(params.polymorphic, relationName);
```

Comment:
for sake of clarity, each call of `polymorphicParams()` now has the same exact signature for each relation setup function